dynamic_movement_primitive/TransformationSystemMsg transformation_system
dynamic_movement_primitive/ICRA2009TransformationSystemStateMsg[] states
dynamic_movement_primitive/ICRA2009TransformationSystemParametersMsg[] parameters
